daemon/cmd/kube_proxy_replacement.go

```diff
@@ -283,6 +283,16 @@ func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) {
 		return
 	}
 
+	if option.Config.EnableLoadBalancerSourceRangeCheck && !probe.HaveFullLPM() {
+		msg := "LoadBalancer source range check requires kernel 4.16 or newer."
+		if isKubeProxyReplacementStrict {
+			log.Fatal(msg)
```

Comment:
Hrm. This is not particularly user-friendly because previous versions would have worked "fine", but now we have introduced a new flag, true by default, which could cause Cilium to crash out upon upgrade. At the very minimum, this should be documented in the upgrade guide. (Maybe that's coming, I'm only part way through the review)

Reply:
This panic is very unlikely to happen, as we panic only if `--kube-proxy-replacement=strict` which requires >= 4.19 kernel, while the full LPM support >= 4.16.